scripts/orchestration/creative_spec_patch_admission_contract.py

```
@@ -0,0 +1,1274 @@
+candidates, mutate runtime truth, use semantic cache, or write graph truth.
+"""
+
+from __future__ import annotations
+
+from collections.abc import Mapping, Sequence
+from pathlib import PurePosixPath
+import re
+from typing import Any, cast
+
+from core.evidence.fingerprints import build_asset_id, build_idempotency_key, fingerprint_payload
+from scripts.orchestration.creative_code_patch_contract import (
```

Flake8 (7.3.0)

[error] 'scripts.orchestration.creative_code_patch_contract.CreativeCodePatchContractError' imported but unused

(F401)